docs/source/install/ansible.rst

```diff
@@ -20,6 +20,7 @@ Our Ansible playbooks support the same platforms as manual installation, i.e.:
 
 * Ubuntu Xenial (16.04)
 * Ubuntu Bionic (18.04)
+* Ubuntu Focal (20.04)
```

Comment:
I'd remove Focal from here, we haven't written the playbooks yet to install via Focal. So until then don't list it here.